### DISCUSSION TITLE
Apply for Maintainer

### DISCUSSION THREAD

**Discussion by Snailedlt** [devicons/devicon#1436] [General]
# Hi everyone! 👋

We are in need of more active maintainers who can take Devicons to the next level 🚀

[Devicons has grown a lot recently](https://star-history.com/#devicons/devicon), and it has grown into an active community of contributors. As such we are in need of more people to review pull requests, label issues and pull requests, update documentation and help guide new contributors.
If you are someone who would like to help out with such things, please go ahead and apply below (one comment per user)!

## ✔️ Maintainer responsibilities

- Review Pull Requests
- Label Issues and Pull Requests
- Close Inactive and/or Duplicate Issues and Pull Requests
- Guide New Contributors

## 📝 How to apply

We suggest you follow the layout\* below:
> \**You can take a look at [my application](https://github.com/devicons/devicon/discussions/1263#discussion-4207441) as a reference.*

**> ABOUT ME**
Tell us about yourself, what are your interests, what skills do you have, and what do you enjoy the most.

**> ME AND DEVICONS**
Let us know why you are a good fit for Devicons, and how you can contribute to Devicons as a maintainer. Have you made any contributions to Devicons yet? If so, tell us about them too, and link to some of the most important ones.

**> WHY I WANT TO JOIN**
Pretty straight forward. Why would you like to join Devicons? Is it to learn, contribute to open-source, add value to your own projects, or something else?

**> MY SOCIALS**
Links to your contact information and/or socials. Pretty much anything that's open and let us contact you in private messages/e-mails. Please also include your discord username/handle so that we can give you the right roles in the discord server.

## 👨‍💻 In which other ways can you contribute?

There are a lot of different stuff to do in Devicons:

- Improve and build out [our website](https://devicon.dev) (`HTML`, `CSS`, `JavaScript`, `Angular`)
- Automate processes (`GitHub Actions`, `CI/CD`, `Python`, `Selenium`, `Gulp.js`)
- Add Icons (`SVG` and `JSON` editing)
- Add more tests to increase the projects robustness and quality (maybe even implement a test framework from scratch)
- Update Our documentation (`GitHub Wiki`, `Markdown`, `HTML`, `CSS`)

**PS:**
We are eligible for [hacktoberfest](https://hacktoberfest.com/), so your maintainer activity and contributions to [Devicons/Devicon](https://github.com/devicons/devicon) through the month of October count towards your hacktoberfest score and rewards.

**PPS:**
Don't dicuss here, use this issue for any questions or discussions: https://github.com/devicons/devicon/issues/1471

**Comment by KayleeDavisGitHub** [comment]
## ABOUT ME
Hello! I'm Kaylee - a data scientist and ML/AI researcher. I mostly program in R, Python, LaTeX, and Markdown to generate results or models and to talk about them through technical writing! I also do "data science for good" by helping out various non-profits and tutoring students in these topics.

## ME AND DEVICONS
I originally found Devicons because I wanted to get images to talk about different programs that I use in [my README](https://github.com/KayleeDavisGitHub/KayleeDavisGitHub) and in various educational documentation I may write. I'm currently writing a [data science book](https://github.com/KayleeDavisGitHub/Graduate_Methods_Handbook/blob/master/Methods.Handbook.pdf) and I plan to reference many of the languages and tools here to supplement some of the data science tools discussed there. Although I haven't pulled, forked, or submitted any new content to Devicons I have taken time to review all of the documentation here, and have even filled out some quick issues to try and add more! (#1444, #1445, and #1446)

## WHY I WANT TO JOIN
My background in data science, data servers, video editing, and content creation may help a lot when identifying which icons are within scope or out of scope for Devicons! I see myself helping a lot with documentation, adding icons, and helping manage issues. Finally, I'm around a lot so I can stay on top of stuff pretty well and help the team out! I am also very experienced at Discord if anyone needs help there, I have moderated and created over five different channels in the past, some of over 1,000 members. 

While in the above I can help, I cannot (currently!) help all that much with web design, automation, and some of the current processes I may have to learn. If that is okay, then I'm here anytime! :)

## MY SOCIALS
GitHub:    https://github.com/KayleeDavisGitHub
LinkedIn: https://www.linkedin.com/in/kayleedavisin/

Feel free to message me in either place if you want to connect or know more! Happy to share my Discord as well 🧡

**Reply by Snailedlt** [reply]
Hi Kaylee! 👋 
First of all, thanks for applying, and thank you for your contributions so far! 🙇 

I see you have a strong background in data science. Your knowledge with python may come in handy for future contributions, since it's what we use for automation. Your willingness to document and help out with the discord server is also nice to see! :)

That said I don't think you fit the bill for a maintainer at the moment. The main reason being the lack of contributions so far. 

I hope you will continue to contribute, and that we can take up this discussion at a later date, when you have a bit more experience with devicons! Don't let this discourage you from coming with suggestions, reviewing PR's and issues, helping others and contributing in whatever way you can. We are always open to new ideas and improvements, be it big or small! There aren't many contributors with data science backgrounds, so your opinions would definitely help see devicons from a different perspective, and hopefully change it for the better 🚀 

**PS:**
You should contact @Panquesito7 about discord, since he's the owner of the discord server.

**Reply by KayleeDavisGitHub** [reply]
@Snailedlt Makes sense, Thank you :)

**Reply by Uddeshya08** [reply]
Hi @Snailedlt @KayleeDavisGitHub I would like to contribute to this and also apply for maintainer.

**Reply by Panquesito7** [reply]
> Hi @Snailedlt @KayleeDavisGitHub I would like to contribute to this and also apply for maintainer.

Hey there! You can check the first message at the top for how to properly apply and get involved in our community. 🙂

**Comment by lunatic-fox** [comment]
Hello there!
I'd like to apply for a maintainer role here at Devicons project.

## ABOUT ME
I'm Josélio Júnior (nickname: Lunatic Fox), from Brazil.
I'm a hobbyist developer and illustrator, and I'm also graduating in Nutrition.
The programming languages that I am more used to are: JavaScript, TypeScript and Lua. I'm also starting to learn C#.
I'm the developer of Devicon API, an derivative API project in TypeScript to serve Devicon plain icons in different colors and sizes.
*Link to project repository:* https://github.com/lunatic-fox/deviconapi

## ME AND DEVICONS
I joined this community because I wanted to add some icons into one of my website's projects, but the icons need to fit some specific themes, so SVG icons are the best choice.
*Link to website's project:* https://lunaticfox.vercel.app/githubLanguages

My Devicons contributions so far:
- #1454 
- #1408
- #1413 
- #1412 
- #1401 
- #1402 
- #1393 
- #1397 
- #1392 
- #1396 
- #1400 

## WHY I WANT TO JOIN
I'd like to join as a maintainer, because I want to help, in my extra time, into Request and PR reviews, and website features.
I'm more active here on GitHub.

## SOCIAL MEDIA
GitHub: https://github.com/lunatic-fox
LinkedIn: https://www.linkedin.com/in/jos%C3%A9lio-j%C3%BAnior-a09b6824b/
Twitter: https://twitter.com/Jojo89373534

**Reply by Snailedlt** [reply]
Hi Josélio! 👋 
Thank you for applying! 

I can vouch for you as a maintainer candidate. You've had plenty of contributions already, and I've found your PR's clean and easy to review. You're quick to fix mistakes, and good at critical thinking, which is essential for reviewing anything. I've also looked at the API you created, and it looks neat! Can't wait for it to support automatic theme detection, will definitely start using it after that's a thing!

We definitely need help with both reviewing and with the website, more reviewers being the main reason we're looking for more maintainers. If you want, we also need help with labeling, closing, linking and otherwise reviewing issues.

@Panquesito7 What do you think?

**Reply by lunatic-fox** [reply]
Thank you very much! 🤩
I'll keep doing my best to this project. 🚀
And I'll take a look at the other tasks.

---

> About the API I have some bad and good news.
> * Bad news: As far as I could see there is no way to `.svg` automatically detect GitHub's theme. Instead, it can only detect when browser theme is changed. 🤔
> * Good news: I made a [page interface](https://lunaticfox.vercel.app/deviconApi) (it's still quite simple) to this project in order to make it a little bit easy. So anyone can search for the icon and simply copy the generated code.

**Reply by Snailedlt** [reply]
Great! thanks for helping 🙇 

> Bad news: As far as I could see there is no way to .svg automatically detect GitHub's theme. Instead, it can only detect when browser theme is changed. 🤔

This is how you change between dark and light mode in markdown: 
```md
![Fancy logo](./dark.png#gh-dark-mode-only)
![Fancy logo](./light.png#gh-light-mode-only)
```
So maybe you could have both the dark-mode and light-mode icons on the same endpoint, and just have separate html id's for them? So the Id's for dark- and light-mode would be `#gh-dark-mode-only` and `#gh-light-mode-only` respectively. Also, maybe we should make an issue/discussion on your repo about this instead ;)

**Reply by Snailedlt** [reply]
@lunatic-fox Great news! I discussed with @Panquesito7, and we both agree you would be a great maintainer. However it seems I have limited permissions to the GitHub organization, so we need to wait an organization owner/admin to give you the role. In the meantime I gave you write access, so you can at least review PRs and stuff :)

Welcome on board!

**Comment by ConX** [comment]
## ABOUT ME
Hi everyone, I am Constantinos, currently living in Austin, TX, USA. I am a data scientist with a computer science background and several contributions to open-source projects. Outside computer science, I also have experience in Adobe Illustrator, CI/CD processes, task automation, and Selenium.

## ME AND DEVICONS
I only recently found Devicons when building the new version of [my website](https://xanthopoulos.info/), specifically the "Technology Stack" section. I liked the simplicity of use and quality of the icons. For my website, there were a few icons that I needed that weren't available and some others that were available only on the `develop` branch. I have already submitted two PRs, and I also submitted an issue I found when trying to build the `develop` version. My current contributions are:

  - #1613
  - #1616
  - #1617
  - #1618
  - #1623
  - #1629

## WHY I WANT TO JOIN
I like contributing to open-source projects. For the Devicons project, I would like to help reduce the backlog of open issues and PRs. Additionally, I would like to contribute by creating requested icons and also towards further automating some of the tasks through Github actions.

## MY SOCIALS
LinkedIn: https://www.linkedin.com/in/conx/

**Reply by Snailedlt** [reply]
Hi Constantinos!
Thank you for applying!

After reviewing a few of your PR's I can vouch for your expertice and willingness to learn! I'm looking forward to seeing more PR's, Issues and other contributions from you!

Just like before we still need tons of help with reviewing, automation, the website, and shrinking our backlog! I think your Selenium, CI/CD and Adobe Illustrator will come in very handy here! Maybe you'll also see some applications for machine learning that the project can benefit from. All in all, I can vouch for you 💯

@lunatic-fox @Panquesito7 What do you guys think?

**Reply by lunatic-fox** [reply]
Hello there Constantinos!
I agree with @Snailedlt and I also think that you'll be a valuable addition to the team!

**Reply by Panquesito7** [reply]
Hey there, Constantinos! 👋

You are welcome to join the team! Your skills will help Devicon grow bigger and better. 🎉
**EDIT:** Let me re-discuss this with the staff and see what they think. 😃

Thanks for your interest in applying and for all your great contributions. 🚀

**Comment by ophilbinbriscoe** [comment]
# ABOUT ME
- 3+ years leading web and software teams & projects at an established startup 👨‍💻
- 10+ years doing freelance design, branding, and communications work
- 10+ years working in tech or related fields (NGO's, gamedev, etc.)
- Bachelor's in Computer Science 🎓
- From a family of graphic designers 👪

# ME AND DEVICONS
I found out about devicons and since then have been building it into projects that I own or maintain. The next release of one of my projects will leverage devicons.

# WHY I WANT TO JOIN
I'd love to be able to review PR's and create releases featuring icon updates or additions. I've noticed a growing backlog of PR's and a slowing of release activity in 2023.

# MY SOCIALS
[LinkedIn](https://www.linkedin.com/in/oliver-anthony/)

**Reply by Panquesito7** [reply]
Hey there! Thanks for your interest in being a maintainer for Devicons! It is greatly appreciated. 🙂
You seem pretty experienced in the design field, and that is awesome for this project.

However, I think you lack contributions to the repository, so, in order to get accepted, I believe you need a few valuable contributions to the project, then we will think about adding you to the team. :)
Thank you for your interest. We'll check your application again once you contribute more to the repository.

Have a nice day ahead! 👋

**Reply by Snailedlt** [reply]
Hi Oliver,
Thank you for applying!

Do you have any links to the projects you've used devicons in? I'd be interested in knowing your usecases. It's always valuable to see the user side of things, so it definitely helps towards becoming a maintainer as well.

Like @Panquesito7 said, I agree that we need some contributions before we can consider you as a maintainer. That said, these contributions can be in the form of making issues, reviewing PR's, making PR's and discussing ideas. Anything helps, and anything that helps gets you a step closer to becoming a maintainer :)

Thanks again for the application! Looking forward to hearing more from you 🚀

**Reply by Snailedlt** [reply]
@ophilbinbriscoe any updates here? 😊

**Comment by weh** [comment]
## ABOUT ME
Hi, i am Patrik, senior software engineer from Switzerland.
I work in the cyber security field, and am into different topics like programming languages in general,
DevOps, Kubernetes, Pentesting,...

I have worked on a Project including SVG image manipulation and combination.

## ME AND DEVICONS
I found the Devicons project while looking for Hacktoberfest contributions, and for me it was a good match.
Some of my contributions include [anaconda, argocd, rust, scala, splunk, llvm, wasm, okta, podman,.. ](https://github.com/devicons/devicon/pulls?q=is%3Apr+author%3A%40me+is%3Aclosed)


## WHY I WANT TO JOIN
The Hacktoberfest idea of giving something back to the community is a great thing.
Allthough i do not participate much during the Year, i still try to contribute sometimes.
This is why i also wannt to help in this project as maintainer.
Maybe with PR Reviews and Icon contributions, or other Tasks that need help.

## MY SOCIALS
Links to your contact information and/or socials. Pretty much anything that's open and let us contact you in private messages/e-mails.
- Github: https://github.com/weh
- Linkedin https://www.linkedin.com/in/patrikwehrli/

**Reply by Snailedlt** [reply]
Thanks for the application, glad to see you here again! :D
Your PR's during the Hacktoberfest were very good, so I can definitely vouch for you! We're also in great need of new maintainers since none of us are particularily active nowadays.

@Panquesito7 @lunatic-fox what do you guys think?

**Reply by lunatic-fox** [reply]
I agree with @Snailedlt. And @weh, I think it will be great to have you as a maintainer.

**Reply by Panquesito7** [reply]
Sounds great! We can actually start by giving triage access. Thank you for your interest and contributions! 🙂

**Reply by Snailedlt** [reply]
@weh We added you as a triager, so you should be able to review PRs and Issues now. Could you please send us your discord username/handle so we can give you the right roles on Discord too? :)

**Comment by Uddeshya08** [comment]
**ABOUT ME**
Hi, I am Uddeshya, Senior Software Developer from India.
I work as a full stack developer and am interested in frontend development.
Even though I have experience of building an application from scratch and deploying it to production. I have worked on angular, react, Java Spring boot, SQL and Azure/AWS and GCP as well.


**ME AND DEVICONS**
I found the Devicon project while I was creating the bio for my Github profile. Also, I was looking for a open source projects like Devicon which makes easy for developers to use icons easily.

**WHY I WANT TO JOIN**
I think as a maintainer at Devicon I would be able to contribute my all skills and learn a lot from this open-source projects. Icons are essential part of any project, and I would use my knowledge and skills to make Devicon better product and reachable to wider audience. Also, I am always keen to learn and like problem solving and finding and fixing bugs. Documentation has always been my favorite part, so looking forward to contributing here too.
 achieve the above tasks.

**MY SOCIALS**
Links to your contact information and/or socials. Pretty much anything that's open and let us contact you in private messages/e-mails.

Github: https://github.com/Uddeshya08
Linkedin: https://www.linkedin.com/in/uddeshyasharma

**Reply by Snailedlt** [reply]
Hi Uddeshya!
Thank you for applying to the maintainer role!
Do you have any links to contributions to devicons or other projects you've worked on?

**Comment by canaleal** [comment]
## ABOUT ME
Hey there, I’m Alex, a junior full-stack software developer from Canada. Although most of my time is spent in front end development, backend development, and automation, I’m always trying new things and contributing wherever possible. 

## ME AND DEVICONS
Even though I’ve been using Devicons for almost 2 years now, ever since Release 2.14.0 back in Sep 9, 2021, most of my contributions are quite recent. Note that due to a limited number of maintainers (One of the reasons I want to join), not all PRs have been merged but many have been reviewed already. Regardless, some of my contributions Include: 


### Icons:
- [update icon: svelte-original.svg (original)](https://github.com/devicons/devicon/pull/1201)
- [new icon: streamlit (original, plain, original-wordmark, plain-wordmark)](https://github.com/devicons/devicon/pull/1207)
- [new icon: postcss (original, plain, original-wordmark, plain-wordmark)](https://github.com/devicons/devicon/pull/1212)
- [new icon: Pulumi (original, original-wordmark, plain, plain-wordmark)](https://github.com/devicons/devicon/pull/1882) 
- [new icon: RxJS (original, plain)](https://github.com/devicons/devicon/pull/1883)
- [new icon: rockylinux (original, original-wordmark, plain, plain-wordmark)](https://github.com/devicons/devicon/pull/1884) 
- [new icon: ory (original, original-wordmark, plain, plain-wordmark)](https://github.com/devicons/devicon/pull/1885)
- [new icon: almalinux (original, original-wordmark, plain, plain-wordmark)](https://github.com/devicons/devicon/pull/1889)
- [new icon: ngrok (original)](https://github.com/devicons/devicon/pull/1891)

### Wiki, Issue Suggestions and PR Reviews: 
- https://github.com/devicons/devicon/issues?q=commenter%3Acanaleal++is%3Aissue
- https://github.com/devicons/devicon/pulls?q=commenter%3Acanaleal++is%3Apr 
- https://github.com/devicons/devicon/wiki/Updating-An-Icon (Updated Links)

### Personal Projects that use Devicon:
- https://alexcanales.vercel.app/
- https://devicon-ui-canaleal.vercel.app/ 

The devicon ui project is my own version of the Devicon website that includes icons from the master branch (Release 2.15.1) as well as all icons currently in the development branch. The site includes searching, filtering by icon version types, tags, and even includes an editor similar to Font Awesome. 

## WHY I WANT TO JOIN
As mentioned previously, I’ve been using Devicon for almost 2 years now. From personal projects to production, I’ve found that Devicon has provided the largest number of programming related icons while also maintaining a level of quality that is unmatched by other projects. I wish to contribute to this project, not just by adding new icons, but actively engaging with the community by providing PR Reviews, answering questions, maintaining issues, fixing any bugs and hopefully assisting in the creation of the new devicon website. 

## MY SOCIALS
You can find me on the Devicon Discrod, but feel free to contact me anywhere:
- Github:[ https://github.com/canaleal](https://github.com/canaleal)
- Linkedin https://www.linkedin.com/in/alex-canales/

**Reply by Snailedlt** [reply]
Hi Alex!
Thank you very much for all your contributions and for deciding to apply to the maintainer role!

I haven't been very active lately, but I've seen some of your PRs which have been of high quality. I see you've also helped with reviewing PRs which is exactly what we need!

For me this is a definite yes!
@lunatic-fox @Panquesito7 what do you guys think?

**Reply by Panquesito7** [reply]
I completely agree with @Snailedlt. Thank you for your awesome contributions.
I haven't been active lately as well, but I hope to be 100% back very soon.

We can start by giving you triage access. What do you think? 😉

**Reply by canaleal** [reply]
Sounds good, thank you @Snailedlt and @Panquesito7.

**Reply by Snailedlt** [reply]
@canaleal Our pleasure! Added you as a triager now!
Thank you for all your contributions so far, and the ones yet to come! :100:
Could you please send us your discord username/handle so we can give you the right roles on Discord too? :)

**Reply by canaleal** [reply]
Sure @Snailedlt, it's `dr.alexalbert`. My avatar is a cat with sunglasses.

**Reply by Snailedlt** [reply]
Thanks! Added :)

**Comment by EnisMulic** [comment]
> ABOUT ME

Software Engineer (almost 3 years of professional experience) mostly focused on backend and devops.

> ME AND DEVICONS

Made a couple of contributions several years ago, mostly high-jacking stale PRs and resolving them.

https://github.com/devicons/devicon/commits?author=EnisMulic

> WHY I WANT TO JOIN

Contribute to open-source, add value to my own projects

> MY SOCIALS

Discord Username: enis1

👨‍💻 In which other ways can you contribute?
There are a lot of different stuff to do in Devicons:


* Automate processes (GitHub Actions, CI/CD, Python, Selenium, Gulp.js)
* Add Icons (SVG and JSON editing)
* Add more tests to increase the projects robustness and quality (maybe even implement a test framework from scratch)
* Update Our documentation (GitHub Wiki, Markdown, HTML, CSS)

**Comment by dinbtechit** [comment]
# ABOUT ME

I'm a Software Developer/UX Designer. I love automating development flow and building open source tools to improve development experience. 

# ME AND DEVICONS

So far, I have not made any code contributions to devicons but I've submitted issues to requesting for new icons. But I use deviocons on my resume and personal projects. The fact that this project is available as a npm package is a main reason I use this. I find it super useful. However, requesting new and updating icons is a bit tedious.  Below are some of my notable open-source repositories that I maintain.

- Jetbrains - VSCode theme ([![Downloads](https://img.shields.io/jetbrains/plugin/d/19177.svg)](https://plugins.jetbrains.com/plugin/19177)) https://github.com/dinbtechit/vscode-theme
- Jetbrains - NestJS ([![Downloads](https://img.shields.io/jetbrains/plugin/d/22771-nestjs.svg)](https://plugins.jetbrains.com/plugin/22771-nestjs)) https://github.com/dinbtechit/jetbrains-nestjs
-  Jestbrains - Ngxs ([![Downloads](https://img.shields.io/jetbrains/plugin/d/22604-ngxs.svg)](https://plugins.jetbrains.com/plugin/22604-ngxs)) - https://github.com/dinbtechit/ngxs

# WHY I WANT TO JOIN

I would like to fully automate the icon contribution process. Basically would like to build a web app/ online editor to upload the icons, which would automatically commit them in the GitHub repository and perform nightly builds and update the npm package on a nightly basis. 

Currently, the way to request and update existing icons is a little tedious and time-consuming and I would like to improve that.

# MY SOCIALS

- Github: https://github.com/dinbtechit 
- LinkedIn: https://www.linkedin.com/in/dinesh-srinivasan-6a01571a/

**Reply by Snailedlt** [reply]
Hi there @dinbtechit!

Thank you for applying for maintainer!
We don't generally accept applications from people who haven't contributed, so it would very much help your case if you made some contributions to the repo :)

I very much agree with you that the workflow we have now is sub-optimal, and it's something I've wanted to fix myself, but I just haven't found time to do it, so we're very open to PR's or suggestions!

For the time being I think it would be better if you made some contributions  before we make you a maintainer. That can either be a feature (like something on the website or workflow related), or some new/updated icons.

If you wanna start working on the workflow right away, please feel free to do so!
I'll help where I can with advice and insight into the codebase, so if you want you can reach out to me in our discord server :)

**Comment by AssassinK786** [comment]
Any Update who is the maintainer?

**Reply by Snailedlt** [reply]
Many of the applicants in this thread have become maintainers :)

**Reply by AssassinK786** [reply]
Then why repo is not updated?

**Reply by Snailedlt** [reply]
It has been updated since this was posted.
The reason there's not much activity here is because all the maintainers have little time to contribute, which is why we still need more maintainers, and this post is still open

**Comment by ReenigneArcher** [comment]
> ABOUT ME

I'm a degreed Mechanical Engineer, who transitioned to Software Engineering after teaching my self how to program in 2020. I maintain a popular project named "Sunshine" (https://github.com/LizardByte/Sunshine) and 50+ repos in that org. Other's have been minorly popular, but nothing close to how popular Sunshine is. I enjoy problem solving and facing new challenges. The success of Sunshine is tightly coupled with the automations and ci/cd I have put in place across the org which help reduce the burden on the coding side.

> ME AND DEVICONS

I can likely help with just about anything, but mainly GitHub actions, Python, and Selenium related stuff. If any difficult problems come up outside of those categories, I'd be happy to try to solve them as well.

I have the following PRs which are all blocked right now (due to not enough maintainers?). https://github.com/devicons/devicon/issues?q=is%3Apr+is%3Aopen+author%3AReenigneArcher

- https://github.com/devicons/devicon/pull/2310 (Required for any new release of Devicons)
- https://github.com/devicons/devicon/pull/2311 (Will help avoid issues like the above one solves)
- https://github.com/devicons/devicon/pull/2312 (Fixes security issue)
- https://github.com/devicons/devicon/pull/2295 (Allow contributors to submit more than one icon at once, which is tedious)
- https://github.com/devicons/devicon/pull/2270 (Adds ~150 new icons and adds github language names/alias so you can easily map between github api and devicons)

I also have other ideas for improvements to the project, but will probably wait until the above 5 are merged before spending more time on it.

Lastly, I can offer guidance on best practices for repository management... or at least based on my experience from managing Sunshine.

> WHY I WANT TO JOIN

Honestly, it is simply because no one is addressing the above PRs. I really just want #2270 to get merged so I can actually utilize Devicons on our website. The fact that just doing that requires the 2 more of the PRs to get merged indicates this project needs some assistance.

> MY SOCIALS

- https://github.com/ReenigneArcher
- https://reddit.com/u/reenignearcher
- Discord: ReenigneArcher

**Reply by Snailedlt** [reply]
Hi @ReenigneArcher!
You're right, we definitely need more maintainers, especially for reviewing PRs and maintaining workflows.
I for one am very happy with what I've seen from you so far. I started reviewing some of your PRs but sadly haven't had time to finish it.

I'd be happy to have you on the maintainer team!

@ConX @weh @canaleal @Panquesito7 @lunatic-fox what do you guys think? :)

**Reply by canaleal** [reply]
I agree with **@Snailedlt**.  

**@ReenigneArcher** has consistently shown that they have the skills to write clean, solid code and the experience to back it up. They’re quick, reliable, and always willing to jump in to help. On top of that, their feedback is thoughtful and constructive, making collaboration easier for everyone.

Massive bonus points for taking the time to look at and fix the deprecated actions issue!

**Reply by Snailedlt** [reply]
Well, since there have been no objections yet, and since you have made significant contributions to the project we'll gladly give you the triager role.

Welcome on board @ReenigneArcher! 🎉

**Comment by Yashh56** [comment]
# *ABOUT ME*

Hello everyone! I'm Yash from India, currently pursuing a Bachelor’s degree in Computer Science. I’m a full-stack developer with a strong interest in frontend, backend, and cloud technologies. Beyond coding, I enjoy watching anime, reading books, and playing games, which help me stay creative and balanced. I also love exploring new ideas and tools to broaden my skills and improve as a developer.


# *ME AND DEVICONS*


Even though I’ve been using *Devicons* in many of my projects, I started contributing around October 2023 during Hacktoberfest to become more actively involved in the open-source ecosystem. 

### Personal Project that uses icons from Devicons:
- https://yashh56.me

### Icons:

- https://github.com/devicons/devicon/pull/2196
-  https://github.com/devicons/devicon/pull/1938
-  https://github.com/devicons/devicon/pull/1920
-  https://github.com/devicons/devicon/pull/1912
-  https://github.com/devicons/devicon/issues/2151
-  https://github.com/devicons/devicon/pull/2548
-  https://github.com/devicons/devicon/pull/2549
-  https://github.com/devicons/devicon/pull/2596 
-  https://github.com/devicons/devicon/pull/2603
-  https://github.com/devicons/devicon/pull/2611
- https://github.com/devicons/devicon/pull/2665


# *WHY  I WANT TO JOIN*

In the last two years, I’ve gained hands-on experience with SVG files and Inkscape while contributing to *Devicon*.
I’d love to contribute to *Devicons* not just by adding new icons, but also by collaborating with the community, reviewing and managing pull requests, fixing bugs, and maintaining issues.

Additionally, I can work on both the client and server sides of the project to help improve its functionality, performance, and overall developer experience.


# MY SOCIALS

#### Github : https://github.com/Yashh56
#### Linkedin: https://www.linkedin.com/in/yash596/
#### Discord : *itsyashbtw* | *568021861871517716*

**Reply by Snailedlt** [reply]
Welcome on board!
